tests/components/wallbox/test_number.py

```diff
@@ -212,3 +218,99 @@ async def test_wallbox_number_class_platform_not_ready(
     assert state is None
 
     await hass.config_entries.async_unload(entry.entry_id)
+
+
+async def test_wallbox_number_class_icp_energy(
+    hass: HomeAssistant, entry: MockConfigEntry
+) -> None:
+    """Test wallbox sensor class."""
+
+    await setup_integration(hass, entry)
+
+    with requests_mock.Mocker() as mock_request:
+        mock_request.get(
+            "https://user-api.wall-box.com/users/signin",
+            json=authorisation_response,
+            status_code=200,
+        )
+
+        mock_request.post(
+            "https://api.wall-box.com/chargers/config/12345",
+            json=json.loads(json.dumps({CHARGER_MAX_ICP_CURRENT_KEY: 10})),
+            status_code=200,
+        )
+
+        await hass.services.async_call(
+            "number",
```

Comment:
```suggestion
            NUMBER_DOMAIN
```
`from homeassistant.components.number import DOMAIN as NUMBER_DOMAIN

Reply:
ahh, nice, will also do those for the other tests in a sep PR